### PR TITLE
preliminary debian packaging

### PR DESCRIPTION
**Pull request by themusicgod1**
### What was wrong?

eth-utils is not in Debian 

### How was it fixed?

created a Debian package directory that successfully builds a Debian package so Debian at least has something to work with when packaging it.

#### Cute Animal Picture

bird

### more

Ethereum is not in Debian. PyEthereum is a good candidate for remedying that.   PyEthereum requires eth-utils.  As a side effect of getting Ethereum into Debian, it would also get into Ubuntu, Mint, Tails, Devuan, and dozens of other distributions both current and future. On the Debian side, this effort is tracked in the following Request for Package (RFP): https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=882695 and the ethereum effort more broadly is being packaged (ITP) at https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=866167 

I have made a fork and added what seems to be the minimum necessary changes to actually be a Debian package, though since I am not actually a DM/DD/maintainer the name will eventually have to be changed by whoever with the actual credentials required to be a maintainer the Debian project chooses to be the maintainer, but it's a first attempt anyway. It does build and allows for eth-utils to be installed on a Debian GNU/Linux system (ie without pip).

note there was one line of code change required to make it appear to work, discussed in issue #31 https://github.com/ethereum/eth-utils/issues/31

### CODE REVIEW

File: eth_utils/__init__.py [hunk 1]
```diff
@@ -80,4 +80,4 @@
 )
 
 
-__version__ = pkg_resources.get_distribution("eth_utils").version
+# __version__ = pkg_resources.get_distribution("eth_utils").version
```
Comment (pipermerriam) [inline eth_utils/__init__.py]: You can rebase from master at which point this change can be reverted.

**Comment by pipermerriam** [comment]
Can you link me to any resources on how this would need to be built and released?

**Comment by themusicgod1** [comment]
https://www.debian.org/doc/manuals/maint-guide/build.en.html ?

**Comment by pipermerriam** [comment]
I closed this because while I understand this type of packaging may be valuable, it's not something we can currently commit to supporting as a team and I don't want this to *rot*.  Feel free to pursue this independently.

**Pull request closed without merging**